setup.py

```diff
@@ -109,7 +109,8 @@
             "flake8-rst-docstrings>=0.2,<0.4",
             "flake8-use-fstring>=1.0,<1.5",
             "mccabe>=0.6,<0.8",
-            "nbval>=0.9,<0.11",
+            "nbconvert>=7,<8",
+            "nbformat>=5,<6",
```

Comment:
We weren't actually using `nbval` anymore so I removed it. The other two are required for the notebook integration tests I added.

Reply:
Whoops, based on the build failure it looks like some additional jupyter dependency is required to actually run the notebooks though!  Will track it down and add.